src/components/molecules/NavList.tsx

```diff
@@ -1,7 +1,12 @@
-import React from 'react';
-import {List, ListItemButton, ListItemText, ListSubheader} from '@mui/material';
+import React from 'react'
+import {
+  List,
+  ListItemButton,
+  ListItemText,
+  ListSubheader,
+} from '@mui/material'
```

Comment:
Is there a way we can avoid this with prettier? Or is this something you like? I don't like the new lines on the imports here, but technically doesn't matter.

Reply:
I'm not seeing options for this.